plugins/embed-optimizer/class-embed-optimizer-tag-visitor.php

```diff
@@ -283,4 +195,134 @@ private function reduce_layout_shifts( OD_Tag_Visitor_Context $context ): void {
 			$processor->append_head_html( sprintf( "<style>\n%s\n</style>\n", join( "\n", $style_rules ) ) );
 		}
 	}
+
+	/**
+	 * Gets preconnect URLs based on embed type
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param OD_HTML_Tag_Processor $processor Processor.
+	 * @return array<non-empty-string> Array of URLs to preconnect to.
+	 */
+	private function get_preconnect_urls( OD_HTML_Tag_Processor $processor ): array {
+		/*
+		* The following embeds have been chosen for optimization due to their relative popularity among all embed types.
+		* See <https://colab.sandbox.google.com/drive/1nSpg3qoCLY-cBTV2zOUkgUCU7R7X2f_R?resourcekey=0-MgT7Ur0pT__vw-5_AHjgWQ#scrollTo=utZv59sXzXvS>.
```

Comment:
FYI, this link doesn't seem to be publicly accessible.

Reply:
Oh, thanks for that. 

@adamsilverstein Can this be exported to a Gist?